yarn-project/sequencer-client/src/sequencer/metrics.ts

```diff
@@ -36,6 +36,11 @@ export class SequencerMetrics {
   private slots: UpDownCounter;
   private filledSlots: UpDownCounter;
 
+  private fishermanBlockBuildFailed: UpDownCounter;
+  private fishermanBlockBuildSuccess: UpDownCounter;
+  private fishermanPrecheckFailed: UpDownCounter;
+  private fishermanSlashingAttempts: UpDownCounter;
```

Comment:
Why not make these regular metrics, as opposed to fisherman-only? It seems like valuable info to have for regular nodes as well.